src/org/zaproxy/zap/extension/ascanrulesAlpha/ZapAddOn.xml

```diff
@@ -34,10 +35,12 @@
 		<ascanrule>org.zaproxy.zap.extension.ascanrulesAlpha.TestUserAgent</ascanrule>
 		<ascanrule>org.zaproxy.zap.extension.ascanrulesAlpha.TraceAxdScanner</ascanrule>
 		<ascanrule>org.zaproxy.zap.extension.ascanrulesAlpha.HtAccessScanner</ascanrule>
+		<ascanrule>org.zaproxy.zap.extension.ascanrulesAlpha.HiddenFileScanner</ascanrule>
 	</ascanrules>
 	<pscanrules/>
 	<files>
 		<file>xml/example-ascan-file.txt</file>
+		<file>json/config.json</file>

```

Comment:
The file name needs to be more descriptive - users will just see the filename 'config.json' and have no idea what its for. Something like 'hidden-files-ascanrule.json' ?